configs/devnet.json

```diff
@@ -45,5 +45,12 @@
   ],
   "spot_pool_config": {
     "show_apr_tooltip": false
+  },
+  "announcement_banner": {
+    "show_from": "2024-12-17T06:30+00:00",
+    "show_until": "2024-12-17T06:49+00:00",
```

Comment:
add tests for `show_from` and `show_until` to ensure show_until is a later date than show_from in `scripts/check_configs.ts`